x/sequencer/keeper/sequencer.go

```diff
@@ -157,7 +121,7 @@ func (k Keeper) GetMatureUnbondingSequencers(ctx sdk.Context, endTime time.Time)
 	return
 }
 
-func (k Keeper) setUnbondingSequencerQueue(ctx sdk.Context, sequencer types.Sequencer) {
+func (k Keeper) SetUnbondingSequencerQueue(ctx sdk.Context, sequencer types.Sequencer) {
 	store := ctx.KVStore(k.storeKey)
 	b := k.cdc.MustMarshal(&sequencer)
 
```

Comment:
weird function name because it adds the sequencer to the queue, not set the queue, as such